docs/run.md

```diff
@@ -2,16 +2,36 @@
 
 The two basic ways to run the simulation are a Python package and command line interface. Both interfaces take simulation parameters as a configuration file and several other user inputs as function arguments or command line arguments.
```

Comment:
This file needs text line wrapping.

Reply:
Done for the whole file, 80 chars. We don't have anything in place to check for it. Maybe I should add it.